package/kernel/linux/modules/netdevices.mk

```diff
@@ -558,7 +558,7 @@ define KernelPackage/ixgbe
   DEPENDS:=@PCI_SUPPORT +kmod-mdio +kmod-ptp
```

Comment:
This package probably has a dependency to  now

Reply:
> Please compile test this on a target which does not compile kmod-hwmon-core into the kernel.

Not sure I follow:

```
target/linux/x86/generic/config-default:CONFIG_HWMON=y
target/linux/x86/64/config-default:CONFIG_HWMON=y
target/linux/x86/geode/config-default:CONFIG_HWMON=y
target/linux/x86/legacy/config-default:CONFIG_HWMON=y
target/linux/layerscape/armv8_32b/config-4.9:CONFIG_HWMON=y
target/linux/layerscape/armv8_32b/config-4.9:CONFIG_IGB=y
target/linux/layerscape/armv8_64b/config-4.9:CONFIG_HWMON=y
target/linux/layerscape/armv8_64b/config-4.9:CONFIG_IGB=y
```

There's no platform which can use the IGB or IXGBE (that I can tell, anyway) which doesn't already have `kmod-hwmon-core` built-in.

Reply:
@hauke @blogic This didn't fail because of missing dependencies... it failed because it was being built on platforms that were nonsensical.  Only` x86`, `x86_64`, `armv8_32b` and `armv8_64b` should be building `kmod-igb.ko` and `kmog-ixgbe.ko`

What are the correct `DEPENDS` terms to express that?  Obviously `@TARGET_x86||TARGET_x86_64` for x86 hardware...  What's the test for the ARM stuff?